### ISSUE TITLE
Feature Request - Identify between native and browsers.

### ISSUE THREAD

**Issue by eujc21** [patrickml/react-native-event-horizon#1]
Would like to have the package identify between native and broswes, aka global and windows.

**Comment by patrickml** [comment]
@eujc21 agree, I'm wondering if we should rename the repo as well. Sadly the "event-horizon" repo is used for the meteor version.

**Comment by eujc21** [comment]
@patrickml Just name it to event-horizon-js, ?

**Comment by patrickml** [comment]
@eujc21 good idea, i may rename the other one `-meteor` then
